docs/api/package.json

```diff
@@ -11,9 +11,7 @@
   ],
   "requirement": {
     "kibana": {
-      "version": {
-        "min": "7.0.0"
-      }
+      "versions": "\u003e=7.0.0"
```

Comment:
@jfsiii This is automatically encoded during marshalling. I don't think this should cause any issue in the UI.

Reply:
I think you're right.

```
> node -p 'console.log("\u003e=7.0.0")'
>=7.0.0
```

Reply:
@jfsiii Any with this, I just learned a nice new trick on how to quickly execute some js code.